Samples/nuget.config

```diff
@@ -9,9 +9,13 @@
     </packageRestore>
     <packageSources>
         <clear />
-        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-        <add key="CsWinRT" value="https://pkgs.dev.azure.com/microsoft/_packaging/CsWinRT/nuget/v3/index.json" />
-        <add key="MSFTNuget" value="https://pkgs.dev.azure.com/microsoft/_packaging/MSFTNuget/nuget/v3/index.json" />
+        <!--
+            Resort to using this direct reference to nuget.org in case of no access to the internal feed below.
```

Comment:
>Resort to using this direct reference to nuget.org in case of no access to the internal feed below. [](http://example.com/codeflow?start=12&length=99)

Is this comment a hint that the comment at the top of this file is not correct? Is this file truly only used in pipelines and not be anyone locally building samples?

Reply:
This comment is not a hint that the comment at the top of this file is not correct. In case a dev tries to use this file on a local machine, say for testing, then the info in the comment might be handy.